inventory/environ.py

```diff
@@ -95,6 +95,7 @@ def getDefaultVars():
     defaultVars["splunk"]["deployer_included"] = True if os.environ.get('SPLUNK_DEPLOYER_URL', False) else False
     defaultVars["splunk"]["indexer_cluster"] = True if os.environ.get('SPLUNK_CLUSTER_MASTER_URL', False) else False
     defaultVars["splunk"]["search_head_cluster"] = True if os.environ.get('SPLUNK_SEARCH_HEAD_CAPTAIN_URL', False) else False
+    defaultVars["splunk"]["dynamic"] = os.environ.get('SPLUNK_DYNAMIC', False)
```

Comment:
Traditional logic is that if the build location starts with http, we treat the source as remote. How is Dynamic changing this behavior?

Reply:
It doesn't... I forget what I was using it for, but we don't need it.